spec/models/ems_refresh/parsers/kubernetes_spec.rb

```diff
@@ -78,4 +79,53 @@
       end
     end
   end
+
+  describe "parse_volumes" do
+    example_volumes = [{:volume                => RecursiveOpenStruct.new(:name => "example-volume1",
```

Comment:
@zakiva use a different indentation, it should be more readable:

    example_volumes = [
      {
        :volume => RecursiveOpenStruct.new(
          :name => "example-volume1",
          :gitRepo => {:repository => "default-git-repository"}),
        :name                  => "example-volume1",
        :git_repository        => "default-git-repository"
      },
      {
        ...